news/2019-02-05-osu-mapping-olympiad-4-results.md

```diff
@@ -0,0 +1,50 @@
+
+A strong **second place** is awarded to [**Minato Yukina**](https://osu.ppy.sh/users/531253) with a total score of **126**! Minato Yukina's entry tied with Sinnoh's for the best Difficulty Spread, and whilst not placing first in any particular category, put in a strong performance across the board to secure a respectably high finish from the veteran.
+
+Rounding out the podium in **third place** is [**Ascendance**](https://osu.ppy.sh/users/2931883) with a total score of **117.5**! Though he could not repeat his feats from 2017's osu!catch Beatmapping Contest, receiving the highest score for hitsounding and solid marks across all other categories helped the most recent *Elite Mapper* in osu!catch edge ahead of fierce competition.
+
+[Complete results for this content, including the judges' comments for each submission can be found here.](https://docs.google.com/spreadsheets/d/1QTQGLqWne1WeJIu36_lS_AIZ0et4-DJ-OngAVCftA08/edit?usp=sharing)
+
+A huge thank you to everyone who submitted an entry and congratulations to the top three finishers! Their rewards will be allocated as follows (as a reminder, 6 Olympiad Points are needed for one badge level):
+
+- **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge) and the *Elite Mapper* title
+- **Second place**: 3 months of osu!supporter, 4 Olympiad Points
+- **Third place**: 2 months of osu!supporter, 2 Olympiad Points
+
+[A complete listing of all accumulated Olympiad Points can be located here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing) which will be updated shortly after each contest.
+
+Additional thanks to our hard-working judges for this contest: [Absolute Zero](https://osu.ppy.sh/users/4847256), [Nokashi](https://osu.ppy.sh/users/5431196) and [-Plus-](https://osu.ppy.sh/users/829284). Unfortunately -Plus- was unable to complete his judge comments due to real life issues which we hope he is able to resolve soon. Nevertheless, our judges have devoted a great deal of time and effort to ensure the success of this contest, so please show your appreciation!
+
+If you enjoyed the contest and song, assistance with the Ranking process for [Sinnoh's winning entry](https://osu.ppy.sh/beatmapsets/921419#fruits/1924123) would be greatly appreciated! Additionally, [all of the entries can be downloaded here](https://drive.google.com/open?id=1cTCPOygsn21-AHpEWPrUsfEg4f6Kfybk) if you wish to check them out for yourselves!
+
+Hopefully you all enjoyed participating in (or simply following) this contest as much as we had organizing it! If you did, be on the lookout for further Olympiad contests in the future!
+
+For those of you who are still eagerly awaiting the osu!taiko and osu!mania Olympiad contests results, we apologise for the delays. There have been a number of unfortunate real-life situations weighing down our judging teams, but we're hoping to get the results published soon!
```

Comment:
earlier in the post is "real life", here is "real-life". both work, just choose one